docs/development.md

```diff
@@ -4,20 +4,40 @@
 
 The CPI Ruby code has unit tests that can be run as follows.
 
+From the project's root dir:
+
 ```bash
+source .envrc
```

Comment:
I was able to execute the tests from the project root dir with the following commands:
```
direnv allow
./src/bosh_aws_cpi/bin/test-unit
```
@Justin-W do you use https://github.com/direnv/direnv? `source .envrc` shouldn't be required if https://github.com/direnv/direnv is used.

Reply:
No, I don't. I don't recall seeing any reference to it as part of the docs, either.

Reply:
I think the better options here is to update the docs with the info that `direnv` is required because all the projects in bosh use it.  WDYT?

Reply:
Not forcing the use of `direnv` is also a good point here. Most probably people using `direnv` will notice that `source .envrc` isn't needed. That is why I going to approve the pr.

Reply:
> I think the better options here is to update the docs with the info that `direnv` is required because all the projects in bosh use it. WDYT?

I agree that documenting the use of `direnv` would be good. Whether as a suggestion vs a requirement is up to you guys. But as of now, it is an invisible requirement which (if unsatisfied, as it was on my machine) makes the currentl-documented development processes unusable.